app/deepseg.cc

```diff
@@ -631,11 +663,15 @@ int main(int argc, char* argv[]) try {
 		// copy new frame to buffer
 		cap.retrieve(raw);
 		ti.retrns = timestamp();
-		ai.set_input_frame(raw);
 		ti.copyns = timestamp();
 
 		if (raw.rows == 0 || raw.cols == 0) continue; // sanity check
 
+		if ( crop_region.height) {
+			raw((cv::Rect_<int>)crop_region).copyTo(raw);
+		}
+		ai.set_input_frame(raw);
```

Comment:
```suggestion

		ai.set_input_frame(raw);
```

Just a bit more space for visual code structure …